import-export-cli/cmd/importAPIPolicy.go

```diff
@@ -0,0 +1,76 @@
+/*
+*  Copyright (c) WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
As mentioned above, the same license header changes for the year and Inc to LLC.